### ISSUE TITLE
Allow `runOnly` to detect its type

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1775]
I want to be able to do this:

```
const result = await axe.run({ runOnly: 'landmark-unique' })
```

Or more specifically, I'd like `runOnly` to take a string or an array of strings, in addition to the object that we're currently passing it. Axe-core should be able to figure out the type based on what we're passing it. Rule IDs and tags are never the same.

When passing a mix of tags and rule IDs axe-core should throw an error. If something unknown is passed, axe-core should ignore it (in the same way axe-core ignores unknown tags).

**Comment by padmavemulapati** [comment]
```
 await axe.run({ runOnly: 'landmark-unique' })
```
 instead of specifing tags and ruleIds ,  either tags or rules using the above syntax for usage of axe-core.js, will get you the runOnly results.
